static/js/apps/base/components/header_bar/menu_desktop_rich_menu.tsx

```diff
@@ -0,0 +1,74 @@
+/**
```

Comment:
(For a follow-up PR): When scrolling down the page with the menu open, the menu sticks to the top.